lib/util.js

```diff
@@ -241,6 +243,23 @@ function inspectPromise(p) {
 
 
 function formatValue(ctx, value, recurseTimes) {
+
+  if (ctx.showProxy && typeof value === 'object') {
+    const inProxyCache = ctx.proxyCache.has(value);
+    var proxy = inProxyCache ? ctx.proxyCache.get(value) : undefined;
```

Comment:
It is necessary to .has() before .get()?  The latter returns undefined when the entry doesn't exist.